select.go

```diff
@@ -360,8 +364,12 @@ func (s *Select) innerRun(cursorPos, scroll int, top rune) (int, string, error)
 		_, idx := s.list.Items()
 		if idx != list.NotFound {
 			break
+		} else if s.EnterAlwaysReturns {
+			clearScreen(sb)
+			rl.Write([]byte(showCursor))
+			rl.Close()
+			return list.NotFound, cur.Get(), err
```

Comment:
`err` here is nil, if someone checks for an error gets none and try to use the index (-1) to access the result, it will panic.

You can test that using the _examples/custom_prompt and search for something that doesn't exist and pressing enter.

Reply:
Thanks, I've addressed this by returning ErrNotFound